app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,15 @@
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_tag(url: '/signin', class: 'sign_in') do |f|
```

Comment:
You should use names routes rather than hard coded urls. That will make maintenance much easier in the future.